client/src/components/Sub_ProductDetail/Sub_ImageGallery/DefaultView.jsx

```diff
@@ -11,7 +13,10 @@ function DefaultView({ images, currentImageIndex, setCurrentImageIndex }) {
         prev
       </button>
 
-      <img src={images[currentImageIndex].url ? images[currentImageIndex].url.substring(images[currentImageIndex].url.indexOf('http')) : '../../../assets/noProdImg.png'} alt="product" />
+      <img
+        src={url ? url.substring(url.indexOf('http')) : defaultUrl}
```

Comment:
nice use of deconstruction on the url to make this easier to read

Reply:
learned from you